### PR TITLE
feat: access list authorization

### PR DESCRIPTION
**Pull request by Alxandr**
## Related Issue(s)
- #253 

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

File: src/Altinn.ResourceRegistry.Core/Auth/AdminAuthorizationHandler.cs
Comment (TheTechArch) [inline src/Altinn.ResourceRegistry.Core/Auth/AdminAuthorizationHandler.cs:1]: Is this used from core project? Feel it is more related to the .Net app itself. Should it be put in the main project.

File: src/ResourceRegistry/Controllers/AccessListsController.cs [hunk 1]
```diff
@@ -82,10 +100,12 @@ public Task<ActionResult<AccessListInfoDto>> UpdateAccessList(string owner, stri
     /// <param name="identifier">The resource owner-unique identifier</param>
     /// <param name="cancellationToken">The <see cref="CancellationToken"/></param>
     /// <returns>A paginated list of <see cref="AccessListMembershipDto"/></returns>
-    [HttpGet("{owner:required}/{identifier:required}/members")]
+    [HttpGet("{identifier:required}/members")]
     [SwaggerOperation(Tags = ["Access List Members"])]
-    public Task<ActionResult<Paginated<AccessListMembershipDto>>> GetAccessListMembers(string owner, string identifier, CancellationToken cancellationToken = default)
+    public async Task<ActionResult<Paginated<AccessListMembershipDto>>> GetAccessListMembers(string owner, string identifier, CancellationToken cancellationToken = default)
```
Comment (TheTechArch) [inline src/ResourceRegistry/Controllers/AccessListsController.cs:105]: Should this method have authorize?

Comment (Alxandr) [reply src/ResourceRegistry/Controllers/AccessListsController.cs:105]: The class has `[Authorize(Policy = AuthzConstants.POLICY_ACCESS_LIST_READ)]` - which is inherited by all the actions.

File: src/Altinn.ResourceRegistry.Core/Auth/AdminAuthorizationHandler.cs
Comment (Alxandr) [reply src/Altinn.ResourceRegistry.Core/Auth/AdminAuthorizationHandler.cs:1]: Moved the whole auth folder into the web project.

**Pull request merged**